blocksync/blocksync.go

```diff
@@ -76,9 +76,8 @@ type (
 
 		startingHeight    uint64 // block number this node started to synchronise from
 		lastTip           uint64
-		lastTipUpdateTime time.Time
```

Comment:
let's keep this for now
1. so we have a record of the current code/algorithm
2. maybe the new algorithm can utilize this info in the future to further improve